README.md

```diff
@@ -30,10 +39,25 @@ Configuration options are documented in [Configuration.md](https://github.com/gi
 
 Before upgrading, please check [upgrading.md](https://github.com/giantswarm/cert-manager-app/blob/master/docs/upgrading.md).
 
+## Release Process
+
+* Ensure CHANGELOG.md is up to date.
+* Create a new branch with name `master#release#vX.X.X`. Automation will create a release PR.
```

Comment:
release#major/minor/patch is also fine, no specific version needed. Don't mind me if this is copied from a template. :)

Reply:
Was a bit torn on which to go with. I think ultimately minor/major/patch is a better option, I'll change it